mysql_merge/mysql_merger.py

```diff
@@ -359,8 +359,7 @@ def copy_data_to_target(self):
                         if not row:
                             break
                         for key, value in row.items():
-                            if value is None:
-                                row[key] = 'NULL'
+                            row[key] = self._conn.escape(value, self._conn.encoders)
```

Comment:
Does this solve the JSON nulls issue?

Reply:
Sure, there is no more any 'NULL's in INSERT queries.